.github/workflows/build_shadow.yml

```diff
@@ -94,8 +92,9 @@ jobs:
       - name: Install Dependencies (yum-centos-7)
         if: ${{ matrix.package_manager == 'yum-centos-7' }}
         run: |
+          # clang on centos 7 requires gcc and gcc-c++ (libgcc is not sufficient)
```

Comment:
Strange that this dependency exists and that it's not marked as on in the package manager, but ok...

Reply:
Yeah, they didn't mark gcc/gcc-c++ as dependencies until CentOS 8 for some reason.